worker/src/dns.rs

```diff
@@ -0,0 +1,83 @@
+use crate::SPRUCE_DIDWEB;
+use anyhow::{anyhow, Result};
+use chrono::{SecondsFormat, Utc};
+use serde::Deserialize;
+use serde_json::json;
+use ssi::{blakesig::hash_public_key, jwk::JWK, vc::Credential};
+use url::Url;
+use uuid::Uuid;
+
+#[derive(Deserialize, Debug)]
+pub struct DnsResponse {
+    pub Answer: Vec<AnswerResponse>,
```

Comment:
Rust is probably giving you a warning for the capitalised field, you can use `#[serde(rename = "name")]`.

Reply:
Above the field, not the struct ;)